crates/oxc_ast/src/visit.rs

```diff
@@ -468,9 +468,11 @@ pub trait Visit<'a>: Sized {
     }
 
     fn visit_class_body(&mut self, body: &ClassBody<'a>) {
+        self.enter_scope(ScopeFlags::ClassBlock);
```

Comment:
You can't do this. A class doesn't have a scope, we need something else to achieve this.